### PR TITLE
adding a scope for groupedlist footer link styles

### LINKED ISSUE #6218: GroupedList footer sets global styles for .ms-Link
### Please provide a reproduction of the bug in a codepen:
 
Repros on the demo site: https://developer.microsoft.com/en-us/fabric#/components/link

### Bug Report 
 
 
- __Package version(s)__:  6.49.2
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? No
 
Requested priority: Normal
 
Products/sites affected: Demo site/internal site
 
#### Describe the issue:
 
`.ms-Link` has a font size of 12px. It was introduced in https://github.com/OfficeDev/office-ui-fabric-react/commit/50afbc71bc286bf38f617fa588121265be4854a6#diff-40a480fc0eed282889dca270e35ffe9fR20

#### Actual behavior:
 
Overrides the link font size globally.
 
#### Expected behavior:
 
Should only override link font size in that scope.

### PR DESCRIPTION
**Pull request by kenotron**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #6218 
- [x] Include a change request file using `$ npm run change`

#### Description of changes

adding a scope for groupedlist footer link styles

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by micahgodbolt** [comment]
Is the :global part still required here?

File: packages/office-ui-fabric-react/src/components/GroupedList/GroupFooter.styles.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ export const getStyles = (props: IGroupFooterStyleProps): IGroupFooterStyles =>
         position: 'relative',
         padding: '5px 38px',
         selectors: {
-          [`:global(.${classNames.link}`]: {
+          [`& :global(.${classNames.link}`]: {
```
Comment (jordandrako) [inline packages/office-ui-fabric-react/src/components/GroupedList/GroupFooter.styles.ts]: Are the parentheses required here? If so, I think you're missing the closing parentheses.

Comment (kenotron) [reply packages/office-ui-fabric-react/src/components/GroupedList/GroupFooter.styles.ts]: wow it's totally broken from the outset!

**Comment by micahgodbolt** [comment]
can we show a link in the footer to demonstrate this? and/or if this is totally silly and we shouldn't be imparting style on raw JSX children, we just remove it :)

File: packages/office-ui-fabric-react/src/components/GroupedList/GroupFooter.styles.ts [hunk 2]
```diff
@@ -17,7 +17,7 @@ export const getStyles = (props: IGroupFooterStyleProps): IGroupFooterStyles =>
         position: 'relative',
         padding: '5px 38px',
         selectors: {
-          [`:global(.${classNames.link}`]: {
+          [`& :global(.${classNames.link})`]: {
```
Comment (cliffkoh) [inline packages/office-ui-fabric-react/src/components/GroupedList/GroupFooter.styles.ts]: Can we add a screener test for this please :)

**Comment by kenotron** [comment]
i could just remove it... seems silly to dictate something like that here...

**Comment by cliffkoh** [comment]
Actually looking at the original issue, I think it makes more sense to remove it.

File: packages/office-ui-fabric-react/src/components/GroupedList/GroupFooter.styles.ts (hunk 2 above)
Comment (kenotron) [reply packages/office-ui-fabric-react/src/components/GroupedList/GroupFooter.styles.ts]: I'll just remove it then :)

**Pull request merged**